### PR TITLE
CalendyInt

### PR DESCRIPTION
**Pull request by thechintan123**
## Rationale

Why are you making this change? Describe the original problems — technical and/or business oriented, which originally necessitated it.

## Advice for Reviewers & Testing Notes

- Explain how your change should be reviewed, and where reviewers should direct their attention.
- Explain (where relevant) how your change can be tested.

## Screenshots:

- Add a screenshot or two of your changes, where applicable.

## Task Name

- Issue Title

## Linting Checklist
- [ ] No commented code
- [ ] Code Formatted nicely (Prettier)
- [ ] PR your own code before you assign reviewers

### CODE REVIEW

**Comment by thechintan123** [comment]
The changes are following:

Created a new CalendyWidget.vue
Added above component in ContactForm.vue
Added script of Calendly widget in index.html

https://user-images.githubusercontent.com/64998391/88920568-6cd1fe80-d2b0-11ea-9e49-ce5911c4112c.png

File: client/src/views/Contact.vue [hunk 1]
```diff
@@ -1,18 +1,20 @@
 <template>
   <div>
     <h1>Contact</h1>
-    <!-- Calendly Widget -->
+    <CalendyWidget />
```
Comment (QjStout) [inline client/src/views/Contact.vue:4]: Typo: Should be 'Calendly' not 'Calendy'

File: client/src/views/Contact.vue [hunk 2]
```diff
@@ -1,18 +1,20 @@
...
     <!-- Email Form -->
     <ContactForm />
   </div>
 </template>
 
 <script>
 import ContactForm from '../components/ContactForm.vue';
+import CalendyWidget from '../components/CalendyWidget.vue';
```
Comment (QjStout) [inline client/src/views/Contact.vue:12]: Typo: Should be 'Calendly' not 'Calendy'

File: client/src/views/Contact.vue [hunk 3]
```diff
@@ -1,18 +1,20 @@
...
 
 export default {
   components: {
-    ContactForm
+    ContactForm: ContactForm,
+    CalendyWidget: CalendyWidget
```
Comment (QjStout) [inline client/src/views/Contact.vue:17]: Typo: Should be 'Calendly' not 'Calendy'

File: client/src/components/CalendyWidget.vue [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+    </div>
+  </div>
+</template>
+
+<script>
+
+</script>
+
+<style lang="scss" scoped>
+</style>
```
Comment (QjStout) [inline client/src/components/CalendyWidget.vue:21]: File name should be 'CalendlyWidget.vue'

Comment (thechintan123) [reply client/src/components/CalendyWidget.vue:21]: Ok. Will update.

File: client/src/views/Contact.vue (hunk 3 above)
Comment (thechintan123) [reply client/src/views/Contact.vue:17]: Ok. Will update.

File: client/src/views/Contact.vue (hunk 2 above)
Comment (thechintan123) [reply client/src/views/Contact.vue:12]: Ok. Will update.

File: client/src/views/Contact.vue (hunk 1 above)
Comment (thechintan123) [reply client/src/views/Contact.vue:4]: Ok. Will update.

**Comment by thechintan123** [comment]
Hi @QjStout 
Do I have to cancel this request and git push again..?

**Comment by engramar** [comment]
@thechintan123 @QjStout , Just letting you know that your testing went well. Your request to book a meeting using Calendly automatically generated the Zoom invite (updating my gmail calendar as well). All good! I recommend you have your own personal calendly/zoom/google calendar integration as well in case you want to add a 'Book a meeting with me' button in your respective profile pages.   
image

**Comment by chenhe-henry** [comment]
Hi @thechintan123, it looks good. 
Remember to git pull the latest version of master branch. 
And there is conflict in client/src/router/index.js,
image
the route name should be lowercase. After you fix this and the typo mentioned by @QjStout, I'm happy to approve. Thanks.

**Pull request merged**